lib/data/multipleBackendGateway.js

```diff
@@ -134,13 +135,20 @@ const multipleBackendGateway = {
         if (client.clientType === 'scality') {
             return client.get(key, range, reqUids, callback);
         }
+
         if (client.clientType === 'aws_s3') {
-            const stream = client.getObject({
+            const action = client.getObject;
+            const method = action.bind(client);
+            const request = method({
```

Comment:
Ok - we can leave it like that here, I will test if we can get rid of it in the refactor.